docs/resume.md

```
@@ -0,0 +1,238 @@
+updated: 2025-11-29
+tags:
+  - status/evergreen
+  - type/web
+published: false
+aliases:
+  - 이력서
+---
+# 이력서
+김승민
+
+010-6660-3515 | smin2020@icloud.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)